src/BlockTypes/RelatedProducts.php

```diff
@@ -113,6 +113,7 @@ public function render_block( string $content, array $block ) {
 			return $content;
 		}
 
+		// If there are no related products, render nothing.
```

Comment:
What do you think about changing
```php
// If there are no related products, render nothing.
$related_products_ids = $this->get_related_products_ids();

if ( count( $related_products_ids ) < 1 ) {
    return '';
}
return $content;
```
to
```php
// If there are no related products, render nothing.
$related_products_ids = $this->get_related_products_ids();
if ( count( $related_products_ids ) < 1 ) {
    return '';
}

return $content;
```

I believe this increases the readability of this section.

Reply:
Fixed with [ba1e861](https://github.com/woocommerce/woocommerce-blocks/pull/9407/commits/ba1e861b826a58098b7d19cae8314d232421c6dc)